SYCL/Basic/host-task-dependency.cpp

```diff
@@ -98,11 +94,11 @@ template <bool UseSYCL2020HostTask> void Thread1Fn(Context *Ctx) {
         GeneratorAcc[Idx] = Idx;
     };
 
-    CGH.single_task<NameGen<class Gen, UseSYCL2020HostTask>>(GeneratorKernel);
+    CGH.single_task<NameGen<class Gen, true>>(GeneratorKernel);
```

Comment:
Does second template parameter still required in `NameGen`? Can it be removed?

Reply:
I removed it